CHANGELOG.md

```diff
@@ -0,0 +1,135 @@
+## To be released
```

Comment:
@sap1ens  What's the reasoning for that?  I was thinking that if we have it there by default, people won't get it wrong and break the Mobitron release script.